script.js

```diff
@@ -0,0 +1,298 @@
+    state = 'add';
+    key = event.key;
+    keyCode = event.code;
+    keyLower = key.toLowerCase();
+
+    event.preventDefault();
+
+    simpleAndSpecial(state);
+    switchKeyCode(state);
+    (sessionStorage.hasOwnProperty('ctrl') && sessionStorage.hasOwnProperty('alt')) ? replaceLanguage() : undefined;
+})
+
+//------------------------------- keyup ----------------------------------
+window.addEventListener('keyup', (event) => {
+    state = 'remove';
+    key = event.key;
+    keyCode = event.code;
+    keyLower = key.toLowerCase();
+
+    simpleAndSpecial(state);
+    switchKeyCode(state);
+});
+
+//------------------------------- end listener ----------------------------------
+
+function replaceLanguage() {
+    if (sessionStorage.hasOwnProperty('lang')) {
+        (sessionStorage.hasOwnProperty('ctrl') && sessionStorage.hasOwnProperty('alt')) ? ((sessionStorage.getItem('lang') === 'Eng') ? sessionStorage.setItem('lang', 'Rus') : sessionStorage.setItem('lang', 'Eng')) :
+            undefined;
+        window.location.reload();
+
+    } else {
+        (sessionStorage.hasOwnProperty('ctrl') && sessionStorage.hasOwnProperty('alt')) ? ((localStorage.getItem('lang') === 'Eng') ? sessionStorage.setItem('lang', 'Rus') : sessionStorage.setItem('lang', 'Eng')) :
+            undefined;
+        window.location.reload();
+    }
+}
+
+function switchKeyCode(state) {
+
+    switch (keyCode) {
+
+        case 'ShiftLeft':
+            keySide(keyLower, left, state);
+            break;
+        case 'ShiftRight':
+            keySide(keyLower, right, state);
+            break;
+        case 'ControlLeft':
+            keySide(keyLower, left, state);
+            break;
+        case 'ControlRight':
+            keySide(keyLower, right, state);
+            break;
+        case 'AltLeft':
+            keySide(keyLower, left, state);
+            break;
+        case 'AltRight':
+            keySide(keyLower, right, state);
+            break;
+        case 'CapsLock':
+            (state === 'add') ? conditionCapsLock() : undefined;
+            break;
+        case 'Tab':
+            FunctionKey(state);
+            break;
+        case 'Backspace':
+            FunctionKey(state);
+            break;
+        case 'Enter':
+            FunctionKey(state);
+            break;
+        case 'Delete':
+            FunctionKey(state);
+            break;
+        case 'ArrowUp':
+            FunctionKey(state);
+            break;
+        case 'ArrowDown':
+            FunctionKey(state);
+            break;
+        case 'ArrowLeft':
+            FunctionKey(state);
+            break;
+        case 'ArrowRight':
+            FunctionKey(state);
+            break;
+        case 'Space':
+            FunctionKey(state);
+            break;
+        case 'MetaLeft':
+            FunctionKey(state);
+            break;
+    }
+};
+
+//  (де)активация нажатых клавиш - буквы и цифры
+function simpleAndSpecial(state) {
```

Comment:
Please start names of the functions from the verb. It will be more clear what function does.